packages/inscription-view/src/components/widgets/code-editor/ResizableCodeEditor.tsx

```diff
@@ -16,10 +16,11 @@ export type CodeEditorAreaProps = Omit<ResizableCodeEditorProps, 'macro' | 'opti
 type ResizableCodeEditorProps = Omit<CodeEditorProps, 'height' | 'context'> & {
   location: string;
   initHeight?: number;
+  areaRef?: React.RefObject<HTMLOutputElement>;
```

Comment:
This is fine, however I'm not very like this solution, as it tells the resize code editor something about the implementaion.
Hm, is it maybe possible to let the initHeight be a function that reads the height from the area

```tsx
initHeight?: () => number;

initHeight={() => areaRef?.current.offsetHeight ?? 90}
```

Reply:
a test would be great, but I can't reproduce it in standalone ^^

Reply:
yep, but not very easy to test I think... :/

Reply:
well well I could try in neo integration, is it worth?

Reply:
I would say no 🤷‍♂️